integrated_channels/cornerstone/exporters/content_metadata.py

```diff
@@ -53,6 +55,32 @@ def export_force_all_catalogs(self):
         """
         return self.export(force_retrieve_all_catalogs=True)
 
+    def encode_invalid_keys(self, content_key):
```

Comment:
Why is this here when there is something similar in `client.py` above?  This seems like more of a utility function -- maybe it should be moved to `cornerstone/utils.py`

Reply:
good point! Yeah, this should really be in a utils file. 👍